packages/aws-rfdk/lib/deadline/lib/stage.ts

```diff
@@ -154,6 +157,12 @@ export class Stage {
       throw new TypeError(`Expected a string "version" but got: ${typeof version}`);
     }
 
+    // Do minimum supported deadline version check
+    const minimumSupportedVersion = '10.1.9';
```

Comment:
Souldn't we check for `10.1.9.2` here? We state in our documentation that we support this version, but this looks to work with `10.1.9.0`.

Reply:
'2' is the only patch that was released under '10.1.9'. So its safe to compare it with '10.1.9' in the code.